python-client/giskard/commands/cli_server.py

```diff
@@ -115,7 +143,16 @@ def _start(attached=False, version=None):
             volumes={home_volume.name: {"bind": "/home/giskard/datadir", "mode": "rw"}},
         )
     container.start()
-    logger.info(f"Giskard Server {version} started. You can access it at http://localhost:{port}")
+
+    up = _wait_backend_ready(port)
+
+    if up:
+        logger.info(f"Giskard Server {version} started. You can access it at http://localhost:{port}")
+    else:
+        logger.warning(
+            "Giskard backend takes unusually long time to start, please check the logs with `giskard server "
+            "logs backend"
```

Comment:
looks like a missing closing **`** after "backend"

Reply:
or perhaps to make it even more readable you can break the long line before the `giskard server logs backend` command, for example after the first comma